src/stream/src/common/table/state_table.rs

```diff
@@ -1377,6 +1377,22 @@ where
         Ok(stream.map_ok(|(_, row)| row))
     }
 
+    /// Get the row from a state table with only 1 row.
+    pub async fn get_from_one_row_table(&self) -> StreamExecutorResult<Option<OwnedRow>> {
+        self.get_row(None)
```

Comment:
Original impl is like this. But I found simple Agg is as simple as `get_row(None)`..
```rust
let sub_range: &(Bound<OwnedRow>, Bound<OwnedRow>) = &(Unbounded, Unbounded);
let stream = self
    .iter_with_prefix(row::empty(), sub_range, Default::default())
    .await?;
pin_mut!(stream);

if let Some(res) = stream.next().await {
    let value = res?.into_owned_row();
    assert!(stream.next().await.is_none());
    Ok(Some(value))
} else {
    Ok(None)
}
```